index.html

```diff
@@ -289,6 +289,15 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div class="contact">
+        <div class="contact-header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
+        <div class="form">
+          <input type="text" name="name" class="input" placeholder="Full name">
+          <input type="email" name="email" class="input" placeholder="Email address">
+          <textarea class="textarea" name="message" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] Good job setting your inputs to their correct types. Let's improve our validation by adding the `maxlength="30"` attribute to our name input, this way our users won't be able to over-extend and maybe type something else that isn't their name.

- [ ] In a similar manner, we can set `maxlength="500"` attribute to our textarea.

- [ ] We can also add the `required` attribute to all of our inputs, this will ensure that our users fill all of the fields from our form before submitting it.

That would complete most of our validation, which will provide a better user experience and also will help us filter data that might not be useful if it isn't complete :tada: I say most because later on, we will learn how we can improve our validation through JavaScript!